Only the failed SQL statement should be displayed in the CLI instead of all the executed statements

**Is your feature request related to a problem? Please describe.**
When executing a list of statements in one batch in the server, if one of the statements fail, the server returns all the statements that are in the batch and not only the statement that fail. This is not a problem for a few statements. However, if a user uses a SQL script with several statements, then they'll see a verbose error in the client result.

i.e.

I have a file with two statements (the create will fail with a syntax error):
```
$ cat /tmp/test.sql 
drop stream if exists s1;
create stream if not exist s1(id int) with (kafka_topic='s1', value_format='json', partitions=1);
```
When I execute the above file, I get the following error:
```
$ ./ksql --file /tmp/test.sql
line 2:8: no viable alternative at input 'create if'
Statement: drop stream if exists s1;
create if not exists stream s1(id int) with (kafka_topic='s1', value_format='json', partitions=1);
Caused by: line 2:8: no viable alternative at input 'create stream if not exist'
Caused by: org.antlr.v4.runtime.NoViableAltException
```

or take this another example with the same statements but using the CLI `-e` parameter:
```
./ksql -e "drop stream if exists s1; create stream if not exist s1(id int) with (kafka_topic='s1', value_format='json', partitions=1);"
line 1:48: no viable alternative at input 'create stream if not exist'
Statement: drop stream if exists s1; create stream if not exist s1(id int) with (kafka_topic='s1', value_format='json', partitions=1);
Caused by: line 1:48: no viable alternative at input 'create stream if not exist'
Caused by: org.antlr.v4.runtime.NoViableAltException
```

The error shows the right error in the first line, but it then displays all the statements again. If a user has 30 statements, then this line will be too large.

**Describe the solution you'd like**
The error should only include the failed statement, not all of them.

i.e.
```
./ksql -e "drop stream if exists s1; create stream if not exist s1(id int) with (kafka_topic='s1', value_format='json', partitions=1);"
line 1:48: no viable alternative at input 'create stream if not exist'
Statement: create stream if not exist s1(id int) with (kafka_topic='s1', value_format='json', partitions=1);
Caused by: line 1:48: no viable alternative at input 'create stream if not exist'
Caused by: org.antlr.v4.runtime.NoViableAltException
```

**Describe alternatives you've considered**
None

**Additional context**
N/A

--- a/ksqldb-cli/src/test/java/io/confluent/ksql/cli/CliTest.java
+++ b/ksqldb-cli/src/test/java/io/confluent/ksql/cli/CliTest.java
@@ -1217,6 +1217,27 @@ public class CliTest {
   }
 
   @Test
+  public void shouldPrintOnlyFailedStatementFromScriptFile() throws Exception {
+    // Given:
+    final File scriptFile = TMP.newFile("script.sql");
+    Files.write(scriptFile.toPath(), (""
+        + "drop stream if exists s1;\n"
+        + "create stream if not exist s1(id int) with (kafka_topic='s1', value_format='json', "
+        + "partitions=1);\n").getBytes(StandardCharsets.UTF_8));
+
+    // When:
+    localCli.runScript(scriptFile.getPath());
+
+    // Then:
+    final String out = terminal.getOutputString();
+    final String expected = "Statement: create stream if not exist s1(id int) "
+        + "with (kafka_topic='s1', value_format='json', partitions=1);\n"
+        + "Caused by: line 2:22: no viable alternative at input 'create stream if not";
+    assertThat(out, containsString(expected));
+    assertThat(out, not(containsString("drop stream if exists")));
+  }
+
+  @Test
   public void shouldUpdateCommandSequenceNumber() throws Exception {
     // Given:
     final String statementText = "create stream foo;";

--- a/ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java
+++ b/ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java
@@ -47,6 +47,10 @@ public class DefaultKsqlParser implements KsqlParser {
           .map(DefaultKsqlParser::parsedStatement)
           .collect(Collectors.toList());
 
+    } catch (final ParsingException e) {
+      // ParsingException counts lines starting from 1
+      final String failedLine =  sql.split(System.lineSeparator())[e.getLineNumber() - 1];
+      throw new ParseFailedException(e.getMessage(), failedLine, e);
     } catch (final Exception e) {
       throw new ParseFailedException(e.getMessage(), sql, e);
     }

